sovrin_node/server/domain_req_handler.py

```diff
@@ -0,0 +1,346 @@
+import json
+from binascii import unhexlify
+from typing import List
+from hashlib import sha256
+
+from plenum.common.exceptions import InvalidClientRequest, \
+    UnauthorizedClientRequest, UnknownIdentifier
+from plenum.common.constants import TXN_TYPE, TARGET_NYM, RAW, ENC, HASH, \
+    VERKEY, DATA, NAME, VERSION
+from plenum.common.types import f
+from plenum.common.util import check_endpoint_valid
+from plenum.server.domain_req_handler import DomainRequestHandler as PHandler
+from sovrin_common.auth import Authoriser
+from sovrin_common.constants import NYM, ROLE, ATTRIB, ENDPOINT, SCHEMA, \
+    ISSUER_KEY, REF
+from sovrin_common.types import Request
+from stp_core.common.log import getlogger
+from stp_core.network.exceptions import EndpointException
+
+
+logger = getlogger()
+
+
+class DomainReqHandler(PHandler):
```

Comment:
What details do we expose by renaming? It's easy to make a mistake and import a wrong class (from Plenum) if they have the same name.